prep-javascript-loops.tar/prep-javascript-loops/main.js

```diff
@@ -0,0 +1,63 @@
+let i = 0;
+const myArray = [];
```

Comment:
These lines should be inside your `whileLoop1` function

Reply:
![Screenshot (137)](https://github.com/Ln1097/javascript-prep-0923-code-solutions/assets/106450990/13c16eb2-3450-4c37-a04e-5c1863a6f3b3)